Modul_6_Dzhumelya/js/index.js

```diff
@@ -0,0 +1,79 @@
+// Вправа №1 Копія масива 
+// через метод slice()
+
+const arr = ['one', 10, 'name', 5, 'Nastya', 9];
+function arrayCopy (array) {
+    return array.slice();
+}
+const newCopyArr = arrayCopy(arr);
+console.log(newCopyArr);
+
+// Вправа №2 
+const arrTwo = ['The', 'quick', 'brown', 'fox', 'jumps', 'over', 'the', 'lazy', 'dog'];
+
+function arrayJoin (arr, delimite) {
+    return arr.join(delimite);
+}
+const arrNewJoin = arrayJoin(arrTwo, ' '); 
+console.log('"' + arrNewJoin + '"');
+
+//  Вправа №3 
+const arrThree = [NaN, 0, 33, false, -51, '', undefined, 75, null];
+function removeFalsy (arr) {
+    return arr.filter((e) => e);
+}
+const arrNoFalsy = removeFalsy(arrThree);
+console.log(arrNoFalsy);
+
+//  Вправа №4 залишив метод concat()
+function getWord2 () {
+    const arr1 = ['e', null, 'r'];
+    const arr2 = ['g', 'y'];
+    const arrUnite = arr1.concat(arr2);
+
+    arrUnite.unshift('p', 'o');
+    arrUnite.push('m', 'o', 'd', 'e' );
+    arrUnite.splice(2, 0, 'w');
+    arrUnite.splice(6,2);
+    arrUnite[6] = 'c';
+    return arrayJoin(removeFalsy(arrUnite), ' ');
```

Comment:
Щоб отримати суцільне слово як розділювач можна використати пустий рядок
```suggestion
    return arrayJoin(removeFalsy(arrUnite), '');
```